### DISCUSSION TITLE
The parameter logits has invalid values

### DISCUSSION THREAD

**Discussion by BERT202206** [openai/whisper#598] [Show and tell]
when I use the 'base' model I get this error ''The parameter logits has invalid values'' but this run OK in 'Tiny', please help me with this.

**Comment by yyf0910** [comment]
The driver version is too high, and the CUDA version does not match the PyTorch version

**Comment by Tahcop123** [comment]
please tell me how to solve it 
my version: cu111 torch1.8.1
